src/a4/domain/IncomeTaxSpace.java

```diff
@@ -0,0 +1,21 @@
+package a4.domain;
+
+public class IncomeTaxSpace extends BoardSpace {
+	public IncomeTaxSpace() {
+	}
+
+	int value;
+
+	int getValue() {
+		return value;
+	}
+
+	void setValue(int new_value) {
+		value = new_value;
+	}
+
+	void chargeTax() {
+		Die tax_die = new Die(6);
+		int tax_multiplier = tax_die.roll();
```

Comment:
Should we return how much tax is being charged so that we can transfer the money from the player?